lib/bump/shared_helpers.rb

```diff
@@ -50,5 +50,9 @@ def self.in_a_forked_process
       return result unless result.is_a?(Hash) && result[:_error_details]
       raise ChildProcessFailed, result[:_error_details]
     end
+
+    def self.excon_middleware
+      Excon.defaults[:middlewares] + [Excon::Middleware::RedirectFollower]
```

Comment:
It's probably in the load path by this point, but would be good to have an explicit `require excon/middleware/redirect_follower` at the top of the file?